packages/grafana-data/src/transformations/transformers/reduce.ts

```diff
@@ -226,6 +226,9 @@ export function reduceFields(data: DataFrame[], matcher: FieldMatcher, reducerId
             ...field,
             values: [value],
           };
+          if (Array.isArray(value)) {
+            copy.type = FieldType.other;
```

Comment:
most reducers just keep the existing type... but some produce an array (all values/unique values) and these should have the type changed to other